hudi-integ-test/src/main/java/org/apache/hudi/integ/testsuite/HoodieTestSuiteWriter.java

```diff
@@ -76,9 +76,9 @@ public HoodieTestSuiteWriter(JavaSparkContext jsc, Properties props, HoodieTestS
     this.deltaStreamerWrapper = new HoodieDeltaStreamerWrapper(cfg, jsc);
     this.hoodieReadClient = new HoodieReadClient(context, cfg.targetBasePath);
     this.writeConfig = getHoodieClientConfig(cfg, props, schema);
-    if (!cfg.useDeltaStreamer) {
-      this.writeClient = new SparkRDDWriteClient(context, writeConfig);
-    }
+    //if (!cfg.useDeltaStreamer) {
+    this.writeClient = new SparkRDDWriteClient(context, writeConfig);
+    //}
```

Comment:
Just remove the lines without using comments?

Reply:
+1 should avoid keeping commented code around

Reply:
sure. will fix it.